kohese-portal/client/src/components/create-wizard/create-wizard.component.ts

```diff
@@ -14,11 +14,14 @@
  * limitations under the License.
  */
 
+// Angular
+import { Component, OnInit, Input, Optional, Inject, ViewChild, ElementRef} from '@angular/core';
```

Comment:
the Angular 9 API for viewchild is now `@viewChild('child', { static: false}) 'child' !: ElementRef`.
Where !: says that child is not of type ElementRef. This change was migrated automatically from version 8, and reverted during migration in Angular 10. This expression is used for safer type checking, however, it did interfere with many of our variables defined as a viewChild. The artifact expressed here (if not used in the code in this file) is a result of my deletion of that part of the viewChild API since that particular variable is indeed of type ElementRef. I will remove this code on a separate cleanup branch in Angular 11.